bundles/org.openhab.ui/doc/components/oh-map-page.md

```diff
@@ -52,12 +52,12 @@ Displays markers on a map
 </PropBlock>
 <PropBlock type="TEXT" name="tileLayerProvider" label="Provider for the background tiles">
   <PropDescription>
-    The provider of tiles to use for the background of the map. Use one from <a class="external text-color-blue" target="_blank" href="https://leaflet-extras.github.io/leaflet-providers/preview/">Leaflet Providers</a>, Some providers will not work until you set options, like access tokens, in the <code>tileLayerProviderOptions</code> parameter (in Code view). See <a class="external text-color-blue" target="_blank" href="https://github.com/leaflet-extras/leaflet-providers#providers-requiring-registration">here</a> for more info. The default is CartoDB, the variant depending on the dark mode setting.
+    The provider of tiles to use for the background of the map. Use one from <a class="external text-color-blue" target="_blank" href="http://leaflet-extras.github.io/leaflet-providers/preview/">Leaflet Providers</a>, Some providers will not work until you set options, like access tokens, in the <code>tileLayerProviderOptions</code> parameter (in Code view). See <a class="external text-color-blue" target="_blank" href="https://github.com/leaflet-extras/leaflet-providers#providers-requiring-registration">here</a> for more info. The default is CartoDB, the variant depending on the dark mode setting.
```

Comment:
These were arguably "regressions" as I've remarked elsewhere but I've since then understood they were added by a PRs which ignored the "automatically generated" nature of these parts of the docs.
I'd say maybe we can accept the regression and then fix those URLs in the proper way (as in, fixing those in the source JS i.e. https://github.com/openhab/openhab-webui/blob/deb8e195fc106ee4f9a602fe1d9642abcadc50b3/bundles/org.openhab.ui/web/src/assets/definitions/widgets/map/index.js#L22-L25)

Reply:
I will try to provide a PR to fix these URLs in the JS ASAP 👍

Reply:
@ghys See #1704.